openverse_catalog/dags/common/requester.py

```diff
@@ -38,12 +38,15 @@ class DelayedRequester:
     rate limits of APIs.
 
     Optional Arguments:
-    delay:  an integer giving the minimum number of seconds to wait
-            between consecutive requests via the `get` method.
+    delay:   an integer giving the minimum number of seconds to wait
+             between consecutive requests via the `get` method.
+    headers: a dict that will be passed in all requests, unless overridden
+             by kwargs in specific calls to the get method
     """
 
-    def __init__(self, delay=0):
+    def __init__(self, delay=0, headers={}):
```

Comment:
ProviderDataIngester already has `headers`, is there a way of using it in this PR:
https://github.com/WordPress/openverse-catalog/blob/70312d08f0400f835a7fbfc1b64fdf7e7ffea0b1/openverse_catalog/dags/providers/provider_api_scripts/provider_data_ingester.py#L59